presto-main/src/test/java/com/facebook/presto/metadata/AbstractMockMetadata.java

```diff
@@ -415,4 +418,10 @@ public boolean catalogExists(Session session, String catalogName)
     {
         throw new UnsupportedOperationException();
     }
+
+    @Override
+    public Set<ConnectorCapabilities> getConnectorCapabilities(Session session, ConnectorId catalogName)
+    {
+        return emptySet();
```

Comment:
Make this throw `UnsupportedOperationException` since that's the behavior of all the other methods in this abstract class. We want this behavior to avoid accidentally calling it in a test.